src/index.html

```diff
@@ -1,13 +1,410 @@
+                src="./images/photos/1.jpg"
+                alt="img_1"
+              >
+            </div>
+            <div class="grid__item grid__item--desktop-9-12">
+              <img
+                class="gallery__img"
+                src="./images/photos/2.jpg"
+                alt="img_1"
+              >
+            </div>
+          </div>
+          <p class="gallery__img-description">
+            <a
+              href="https://goo.gl/maps/pCBpmbTdXYHUE2W56"
+              class="gallery__img-description gallery__img-description--link"
+            >
+               Marina Palms
+            </a>
+            / North Miami Beach, FL 33162
+          </p>
+        </div>
+
+        <!-- Gallery BLOCK 2:  -->
+        <div class="gallery__photos">
+          <div class="grid">
+            <div class="grid__item grid__item--desktop-1-6">
+              <img
+                class="gallery__img"
+                src="./images/photos/3.jpg"
+                alt="img_1"
+              >
+            </div>
+            <div class="grid__item grid__item--desktop-7-12">
+              <img
+                class="gallery__img"
+                src="./images/photos/4.jpg"
+                alt="img_1"
+              >
+            </div>
+          </div>
+          <p class="gallery__img-description">
+            <a
+              href="https://goo.gl/maps/mhLG2WpFteRLgFFJ7"
+              class="gallery__img-description gallery__img-description--link"
+            >
+              Edition Residens
+            </a>
+            / Miami Beach, FL 33139
+          </p>
+        </div>
+
+        <!-- Gallery BLOCK 3:  -->
+        <div class="gallery__photos">
+          <div class="grid">
+            <div class="grid__item grid__item--desktop-1-4">
+              <img
+                class="gallery__img"
+                src="./images/photos/5.jpg"
+                alt="img_1"
+              >
+            </div>
+            <div class="grid__item grid__item--desktop-5-12">
+              <img
+                class="gallery__img"
+                src="./images/photos/6.jpg"
+                alt="img_1"
+              >
+            </div>
+          </div>
+          <p class="gallery__img-description">
+            <a
+              href="https://goo.gl/maps/TG9HnYeCVson2ffS7"
+              class="gallery__img-description--link"
+            >
+              Faena House
+            </a> / Miami Beach, FL 33140
+          </p>
+        </div>
+
+        <!--BUTTON-->
+        <div class="grid grid--desktop">
+          <div class="grid__item grid__item--6-7">
+            <a href="https://www.facebook.com/Miamicondokings/" class="button button--secondary">
+              View all
+            </a>
+          </div>
+        </div>
+      </section>
+
+      <!-- Testimonials:  -->
+      <h2 id="testimonials" class="page__title">
+        Testimonials
+      </h2>
+
+      <section class="page__section testimonials">
```

Comment:
You must use a heading within a section, as this heading describes the section. Section should have heading, if it doesn't,  we gonna have validation errors.